src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {});
+const promise2 = new Promise((resolve, reject) => {});
```

Comment:
![image](https://user-images.githubusercontent.com/122989585/219873156-32a6d7c8-7ba9-497a-9901-574148813e9d.png)
```suggestion
const promise1 = new Promise((resolve, reject) => {
   ....your code should be here
});
const promise2 = new Promise((resolve, reject) => {
   ....your code should be here
});
```